src/hotspot/share/gc/shenandoah/shenandoahControlThread.hpp

```diff
@@ -122,11 +122,10 @@ class ShenandoahControlThread: public ConcurrentGCThread {
   ~ShenandoahControlThread();
 
   // Handle allocation failure from normal allocation.
```

Comment:
Re "normal allocation" : what is not a normal (i.e. abnormal?) allocation?

Reply:
I believe 'normal' here is a mutator allocation. I can update this comment.